src/components/NamespacesList/index.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import './NamespacesList.css';
+import NavBar from '../NavBar';
+import InformationBar from '../InformationBar';
+import SideNav from '../SideNav';
+import getNamespaces from '../../redux/actions/NamespacesActions';
+import Status from '../Status';
+import timePast from '../../helpers/timeUtility';
+
+class NamespacesListPage extends React.Component {
+  componentDidMount() {
+    const { getNamespaces } = this.props;
+    const { match: { params } } = this.props;
+    getNamespaces(params);
+  }
+
+  render() {
+    const { namespacesList } = this.props;
+    const clusterName = localStorage.getItem('clusterName');
```

Comment:
Something i cant wrap my head around. Where is `localStorage` coming from? and when do you put `clusterName`? Because on your branch when i `concole.log` it brings it but i cant seem to make it work on mine. Please explain it to me. Thank you

Reply:
I have seen how you set it. Never mind

Reply:
@Mubangizi unless you have further feedback kindly review and approve.